Relax/Relax/HistoryItem.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  HistoryItem.swift
+//  Relax
+//
+//  Created by Ryan Arana on 2/9/15.
+//  Copyright (c) 2015 PDX-iOS. All rights reserved.
+//
+
+import Foundation
+
+struct HistoryItem {
```

Comment:
I think that'll be better left for an exercise for the workshoppers. Mostly because I can't stub it out without breaking the build.